lib/public/utilities/userPreferencesStore.js

```diff
@@ -0,0 +1,35 @@
+import { Observable } from '/js/src/index.js';
+
+/**
+ * Model class to store the preferences of the users
+ */
+export class UserPreferencesStore extends Observable {
+    /**
+     * Constructor
+     */
+    constructor() {
+        super();
+
+        this._rawTimestamps = false;
+    }
+
+    /**
+     * States if timestamps must be displayed as raw timestamps or formatted as date
+     *
+     * @return {boolean} true if the timestamps must be displayed as is
+     */
+    get rawTimestamps() {
+        return this._rawTimestamps;
+    }
+
+    /**
+     * Set if the timestamp must be defined as raw timestamps or formatted as date
+     *
+     * @param {boolean} value the new preference
+     */
+    set rawTimestamps(value) {
+        this._rawTimestamps = value;
```

Comment:
In order to catch non-boolean values, it might be worth to use `Boolean(value)` here as well? Similar to the one you used for https://github.com/AliceO2Group/Bookkeeping/blob/main/lib/public/views/Tags/Tags.js#L302